packages/datx-jsonapi-angular/projects/datx-jsonapi-angular/src/lib/Response.ts

```diff
@@ -1,9 +1,9 @@
 import { fetchLink, IResponseSnapshot, Response as PromiseResponse } from '@datx/jsonapi';
+import type { ILink } from '@datx/jsonapi-types';
```

Comment:
I'm not sure about this, because it's not just about those libs, but also about the usage in the project. This is a breaking change, so it might be a good idea to reexport the types within @datx/jsonapi just to avoid it?

Reply:
I just re-exported old interfaces here https://github.com/infinum/datx/pull/1167/files#diff-6e096333c00ec510aa25c3e3e5d8d1c7522d841afb1b993d3b17ce6d932b7b85R45 

`ILink` was not one of them so this is an inprovement